README.md

```diff
@@ -250,6 +250,16 @@ the [Lombok Gradle Plugin](https://plugins.gradle.org/plugin/io.freefair.lombok)
 to delombok your source code before it is passed to the Checker Framework
 for typechecking. This plugin does not support any other use of Lombok.
 
+## Incompatibility with explicitly-specified annotation processors
+
+If you provide a `-processor` option to `javac`,
```

Comment:
Can you add a run-time check for whether a `-processor` command-line argument is present?  That will be less error-prone than expecting users to pay attention to this section of the manual.

An even more user-friendly design would be to have two codepaths in the plugin:  if -processor is present, the plugin augments it; if -processor is not present, the plugin creates a manifest and javac does auto-discovery.